### ISSUE TITLE
[Improvement] Pages

### ISSUE THREAD

**Issue by mitchellholzmann** [Plexis/Plexis#12]
I just noticed your going to allow edits on FAQ/Support pages underneath "Site Support". Wouldn't it be better to just make it "Pages" and let people add custom pages like any other CMS does?

Just my 2 cents, in my opinion a much better choice.

**Comment by wilson212** [comment]
I agree with you... We still havent really touched base on the support side of the cms, but most likely it will be as you mentioned, the admin creates these pages manually from a wysiwyg editor in the acp
